astropy/tests/pytest_doctestplus.py

```diff
@@ -11,11 +11,6 @@
 
 from .output_checker import AstropyOutputChecker, FIX
 
-try:
-    import importlib.machinery as importlib_machinery
-except ImportError:  # Python 2.7
-    importlib_machinery = None
-
```

Comment:
I don't think you need to make this change because this file is getting removed anyway from astropy core (and this will just create conflicts with the PR that is removing these files - https://github.com/astropy/astropy/pull/6606)